actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -22,6 +22,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.authenticated_encrypted_cookie_salt = "authenticated encrypted cookie"
     config.action_dispatch.use_authenticated_cookie_encryption = false
     config.action_dispatch.perform_deep_munge = true
+    config.action_dispatch.embed_expirity_in_the_cookie_value = true
```

Comment:
Shouldn't this be set to true in `load_defaults` instead? Otherwise upgrading apps will still break because of the commented out value in `new_framework_defaults`.